widget/image-map/image-map.scss

```diff
@@ -0,0 +1,16 @@
+.or-image-map-initialized {
+	.or-comment-widget {
+		&__content {
+			&__form {
+				&__user {
+					&__dn-notify {
+						&.option-wrapper {
+					        // showing normally on dn-widget
+					        display: block;
```

Comment:
This is really a dn-widget fix I think (because the comment-widget on which the dn-widget is based doesn't have any such fields in the widget). So I'd put it in the same file as the first fix above. Though the best location for this fix is definitely debatable and there may be better ones. Just don't use this duplicate widget fix here.

Alternatively, you could also fix this in enketo-core by changing how .option-wrapper is targeted in the image-map widget, but I agree that seems like it would be a bit too much work,

Reply:
I see, commit updated